package.json

```diff
@@ -1,12 +1,13 @@
 {
   "name": "vite-project",
   "private": true,
-  "version": "0.0.0",
+  "version": "0.1.0",
```

Comment:
This is only my preference. I like to start a new project with an initial version other than 0.0.0 to indicate some change has already happened.